lua/nvim-treesitter/parsers.lua

```diff
@@ -1131,6 +1131,16 @@ list.linkerscript = {
   maintainers = { "@amaanq" },
 }
 
+list.liquid = {
+  install_info = {
+    url = "https://github.com/hankthetank27/tree-sitter-liquid",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```

Comment:
```suggestion
```
`branch` is only needed if it's not the default branch (regardless of the name).